daemon/daemon_uploads.py

```diff
@@ -309,3 +321,69 @@ async def upload_file_to_S3(
     task.change_progress(task.progress + 15)
     logger.info("File upload confirmed with BlenderKit server")
     return True, ""
+
+
+async def ensure_verificationStatus(
+    session: ClientSession, task: daemon_tasks.Task, metadata_response: dict
+) -> str:
+    """Ensure asset verificationStatus is set to uploaded if needed. Logic is:
```

Comment:
Tady extra pozor, logiku jsem nemenil, zustala jak je. Ale pinguju @vilemduha @PetrDlouhy abyste prosli a potvrdili, ze takhle to chceme. Kdyztak udelam zmeny.